helm/alfresco-content-services/values.yaml

```diff
@@ -12,8 +12,8 @@
 repository:
   replicaCount: 2
   image:
-    repository: alfresco/alfresco-content-repository
-    tag: "6.2.0-A2"
+    repository: quay.io/alfresco/alfresco-content-repository
```

Comment:
The tag is available on docker hub:
https://hub.docker.com/r/alfresco/alfresco-content-repository/tags